docs/de/index.md

```diff
@@ -39,6 +39,51 @@ werden muss.
 Zum Abschluss können verschiedene Verhaltensweisen gewählt und konfiguriert werden, wie z. B. ob Branches, Tags oder
 Pull Requests gebaut werden sollen.
 
+#### JobDSL
+
+Um ein Build Job für ein SCM-Manager Repository mit Hilfe der JobDSL anzulegen, 
+kann folgende Syntax für Mercurial und Git Repositories verwendet werden:
+
+```groovy
+multibranchPipelineJob('heart-of-gold') {
+  branchSources {
+    scmManager {
+      id('spaceships/heart-of-gold')
+      serverUrl('https://scm.hitchhiker.com')
+      credentialsId('my-secret-id')
+      repository('spaceships/heart-of-gold')
+      discoverBranches(true)
+      discoverPullRequest(true)
+      discoverTags(true)
```

Comment:
Isn't default for `discoverTags` = false

Reply:
You are right.